src/api/composition/CompositionCollection.js

```diff
@@ -270,67 +274,75 @@ define([
      *
      * @param {number} oldIndex
      * @param {number} newIndex
-     * @memberof module:openmct.CompositionCollection#
      * @name remove
      */
-    CompositionCollection.prototype.reorder = function (oldIndex, newIndex, skipMutate) {
+    reorder(oldIndex, newIndex, _skipMutate) {
         this.provider.reorder(this.domainObject, oldIndex, newIndex);
-    };
-
+    }
+    /**
+     * Destroy mutationListener
+     */
+    _destroy() {
```

Comment:
Should be private, it just predates ES6, so let's make it a private method while we're in here.

Reply:
[It's being used as a public method here](https://github.com/nasa/openmct/blob/master/src/ui/components/ObjectView.vue#L161), so I'm unable to make this private currently.